test/functional/testlib.bash

```diff
@@ -488,6 +488,12 @@ set_env_variables() { # swupd_function
 	debug_msg "STATEDIR: $STATEDIR"
 	export PATH_PREFIX="$TEST_DIRNAME"/testfs/target-dir
 	debug_msg "PATH_PREFIX: $PATH_PREFIX"
+	export TP_BASE_DIR="$TEST_DIRNAME"/3rd-party
+	debug_msg "TP_BASE_DIR: $TP_BASE_DIR"
+	export TP_BASE_STATEDIR="$STATEDIR"/3rd-party
+	debug_msg "TP_BASE_STATEDIR: $TP_BASE_STATEDIR"
+	export TP_TARGETDIR="$env_name"/testfs/target-dir/"$THIRD_PARTY_BUNDLES_DIR"
+	debug_msg "3rd-party repo target dir: $TP_TARGETDIR"
```

Comment:
Again I'm confused about how moving these variables here make them relative to the current test, they may be relative to the current test file, but a test file may have many tests, so the only way of having variables relative to a test is if the variable is set up from within the setup() once we know the name of the test